lib/ansible/modules/storage/netapp/na_ontap_firewall_policy.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/python
+
+# (c) 2018, NetApp, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: na_ontap_firewall_policy
+short_description: NetApp ONTAP Manage a firewall policy
+version_added: '2.7'
+author: NetApp Ansible Team (ng-ansibleteam@netapp.com)
+description:
+  - Manage a firewall policy for an Ontap Cluster
+extends_documentation_fragment:
+  - netapp.na_ontap
+options:
+  state:
+    description:
+      - Whether to set up a fire policy or not
+    choices: ['present', 'absent']
+    default: present
+  allow_list:
+    description:
+      - A list of IPs and masks to use
+  policy:
+    description:
+      - A policy name for the firewall policy
+    required: true
+  service:
+    description:
+      - The service to apply the policy to
+    choices: ['http', 'https', 'ntp', 'rsh', 'snmp', 'ssh', 'telnet']
+    required: true
+  vserver:
+    description:
+      - The Vserver to apply the policy to.
+    required: true
+  enable:
+    description:
+      - enabled firewall
+    choices: ['enable', 'disable']
+    default: enable
+  logging:
+    description:
+      - enable logging
+    choices: ['enable', 'disable']
+    default: disable
+  node:
+    description:
+      - The node to run the firewall configuration on
+    required: True
+'''
+
+EXAMPLES = """
+    - name: create firewall Policy
+      na_ontap_firewall_policy:
+        state: present
+        allow_list: [1.2.3.4/24,1.3.3.4/24]
```

Comment:
Not sure what you've done in other modules, though allowing this to take `type=list` maybe nicer

Reply:
It is defined as a list:
allow_list=dict(required=False, type="list"),

Or do you mean the example can be simplified?

Reply:
Oh, ignore this